example3/Microsoft.Diagnostics.Metric/MetricSource.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Metric
+{
+    public class MetricSource
```

Comment:
how does MeterListener gets attached to a MeterSource? In tracing, there is the static method ActivitySource.AddListener(listener)

Reply:
In the current code you call the MetricListener.Start() function. I hadn't paid too much attention to where it should show up in the API.

Reply:
I'll admit I am liking Start() better than the static MetricSource.AddListener() API at the moment. Maybe if the mismatch is bugging people we can add ActivityListener.Start() to restore parity : )

Reply:
Otel end users won't care as we'll hide it under.
Start() logically makes sense - as it indicates starting of the listening. I'd still prefer consistency :-)